playwright/test/view-items.test.ts

```diff
@@ -71,3 +108,36 @@ describe('Scenario 6: Item has multiple volumes', () => {
     }
   });
 });
+
+describe('Scenario 7: A user wants to navigate an item by its parts', () => {
```

Comment:
We should standardise, I've gone for researcher or person over user from discussions I've had with UX at Wellcome wanting to move away from that word - usually around conversations like [this](https://medium.com/s/user-friendly/why-im-done-saying-user-user-experience-and-ux-in-2019-4fdfc6b7de23)

Might be worth catching up with @alicerichmond when she's back on what she thinks as these stories should be accessible across the team.